### ISSUE TITLE
Windows 11 DEV 22494.1000 start issue

### ISSUE THREAD

**Issue by jarx88** [pavlobu/deskreen#172]
After update windows to next rs in start program i get only this in start:

![image](https://user-images.githubusercontent.com/31516391/140535954-7aabb2e7-f9c1-4c96-b3f8-f0be2c06f1b6.png)

**Comment by jarx88** [comment]
This project is down ?. I find good alternative SPACEDESK

**Comment by pavlobu** [comment]
Nope, Deskreen is not dead. I'm just the only one core developer at the moment. And this project doesn't currently fit into my tight schedule. So I took a little pause with it for now.

**Comment by flug8** [comment]
Any solution for this bug yet?

**Comment by kilzone** [comment]
The 'issue' is that Microsoft removed *wmic.exe* from Windows 11 as it is deprecated (packages should move to new PowerShell WMI). The issue itself is not caused by Deskreen, but by an included package called 'default-gateway' which is referenced by 'internal-ip' - Deskreen uses the 'internal-ip' package to present the QR-code page with URL or show the error message above if none is found.

Right now I don't have time to see if there is an alternative to 'internal-ip', both 'default-gateway' and 'internal-ip' aren't updated yet, so the easiest solution is to download *wmic.exe* again and copy the files to %WINDIR% - I've setup a repository to do this: https://github.com/kilzone/wmic-windows-11/releases/tag/19044.1415
